src/main/java/nextstep/subway/line/domain/ConnectionType.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.subway.line.domain;
+
+import java.util.function.BiFunction;
+
+public enum ConnectionType {
+    FIRST((current, request) -> SectionConnectManager.connectFirstSection(request)),
+    MIDDLE((current, request) -> SectionConnectManager.connectMiddleSection(current, request)),
+    LAST((current, request) -> SectionConnectManager.connectLastSection(request)),
+    NONE((current, request) -> null);
+
+    private BiFunction<Section, Section, Section> expression;
+
+    ConnectionType(BiFunction<Section, Section, Section> expression) {
+        this.expression = expression;
+    }

```

Comment:
`SectionConnectManager.connectFirstSection`
`SectionConnectManager.connectMiddleSection`
`SectionConnectManager.connectLastSection`
세개 모두 request를 그대로 리턴하고있는데 `BiFunction`을 사용할 필요가 있을까요? 🤔

```suggestion
public enum ConnectionType {
    FIRST((current, request) -> {}),
    MIDDLE(Section::divideSection),
    LAST((current, request) -> {}),
    NONE((current, request) -> {});

    private BiConsumer<Section, Section> expression;

    ConnectionType(BiConsumer<Section, Section> expression) {
        this.expression = expression;
    }

```
`BiConsumer`를 사용하는건 어떨까요? 😀

Reply:
제안해주신 코드가 더 깔끔해보이네요! 수정하겠습니다 ㅎㅎ 좋은 의견 감사합니다~